src/osgPlugins/OpenFlight/ReaderWriterFLT.cpp

```diff
@@ -174,7 +178,7 @@ class ReadExternalsVisitor : public osg::NodeVisitor
             std::string filename = node.getFileName(pos);
 
             // read external
-            osg::ref_ptr<osg::Node> external = osgDB::readRefNodeFile(filename,_options.get());
+            osg::ref_ptr<osg::Node> external = osgDB::readNodeFile(filename,_options.get());
```

Comment:
We should take the 3.6 version of this change if we can. This is one that I am more hesitant on though since readNodeFile might go through a different path in the VRV file loader vs readRefNodeFile

Reply:
I made a comment in the source that this was changed for OSG 3.6.   Folks debugging this will see there's a change.